src/components/action-popover/action-popover.spec.tsx

```diff
@@ -1191,34 +1218,64 @@ describe("ActionPopover", () => {
     });
 
     it("validates the submenu prop", () => {
-      jest.spyOn(global.console, "error").mockImplementation(() => {});
-      ReactDOM.render(
-        <ThemeProvider theme={mintTheme}>
-          <ActionPopover>
-            <ActionPopoverItem
-              submenu={<p>foo</p>}
-              icon="pdf"
-              onClick={jest.fn()}
-            >
-              item
-            </ActionPopoverItem>
-          </ActionPopover>
-        </ThemeProvider>,
-        container
+      expect(() => {
```

Comment:
Same comment/ question as above regarding invariant outputs in the console when running specs